Source/MonoGame.Extended/Animations/Easing/CurveEasing.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Xna.Framework;
+
+namespace MonoGame.Extended.Animations.Easing
+{
+    public class CurveEasing : EasingFunction
+    {
+        public CurveEasing(Curve curve) {
```

Comment:
I don't mean to nitpick but your coding style doesn't fit with the rest of the framework. I find it quite annoying when the brackets are on the same line as the method signature. Obviously I can fix this up after the merge but it's one more thing I'm going to have to do.